package.json

```diff
@@ -218,7 +218,7 @@
     "@metamask/eth-keyring-controller@npm:^13.0.1": "patch:@metamask/eth-keyring-controller@npm%3A13.0.1#~/.yarn/patches/@metamask-eth-keyring-controller-npm-13.0.1-06ff83faad.patch",
     "nonce-tracker@npm:^3.0.0": "patch:nonce-tracker@npm%3A3.0.0#~/.yarn/patches/nonce-tracker-npm-3.0.0-c5e9a93f9d.patch",
     "@trezor/connect-web": "9.0.11",
-    "lavamoat-core@npm:^15.1.1": "patch:lavamoat-core@npm%3A15.1.1#~/.yarn/patches/lavamoat-core-npm-15.1.1-51fbe39988.patch"
+    "lavamoat-core@npm:^15.0.0": "patch:lavamoat-core@npm%3A15.0.0#~/.yarn/patches/lavamoat-core-npm-15.0.0-5f5fd568f9.patch"
```

Comment:
It looks like this patch isn't getting applied (v15.1.1 is used in the lockfile). I think this explains the intermittent policy parse issues we're seeing. That intermittent policy parse issue is what the "Skip policy write step to prevent intermittent build failures" step of the patch is meant to fix

Reply:
As an aside, this is not the first time I've noticed an unused patch go unnoticed. We should find a way to flag that in a CI check. We shouldn't have unused patches.

Reply:
Oh, now the whole patch is gone. The regex optimizations need to be preserved

Reply:
Patching patch files is zero fun. Haha